web/styles/app_components.css

```diff
@@ -1103,6 +1103,15 @@ input.settings_text_input {
                 word-wrap: unset;
             }
 
+            .copy-overlay-message {
+                cursor: pointer;
+                color: hsl(0deg 0% 54% / 70%);
```

Comment:
Where is this color from? It might be useful to have a CSS variable if this is shared with other elements.